service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh

```diff
@@ -49,18 +60,18 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-# The linux user that JupyterLab will be running as. It's important to do some parts of setup in the
+# The non-root linux user that JupyterLab will be running as. It's important to do some parts of setup in the
 # user space, such as setting Terra CLI settings which are persisted in the user's $HOME.
 # This post startup script is not run by the same user.
```

Comment:
Would you mind removing this last comment "This post startup script is not run by the same user." I don't think it adds anything and I think is ambiguous.